proto/internalapi/sensor/collector_iservice.proto

```diff
@@ -3,16 +3,22 @@ syntax = "proto3";
 package sensor;
 
 import "google/protobuf/empty.proto";
-import "internalapi/common/collector_runtime_config.proto";
+import "storage/cluster.proto";
```

Comment:
A previous plan had been to use a collections based approach and an internal API fore this was merged. However, the plan was changed to use the existing cluster config. That is the reason for this change.